connection/journey.test.js

```diff
@@ -1,9 +1,59 @@
-import { AuthUserPasswordCredentials, AuthAccessTokenCredentials } from './auth.js';
```

Comment:
I think we need a few mock-tests for authentication (eg with a mock http server), especially around refresh.

Maybe have a look here: https://github.com/weaviate/weaviate-python-client/blob/main/mock_tests/test_auth.py

Reply:
Done, good suggestion